src/mrchat-private.h

```diff
@@ -61,7 +61,8 @@ int             mrchat_are_all_members_verified__ (mrchat_t*);
 
 
 #define         MR_CHAT_PREFIX              "Chat:"      /* you MUST NOT modify this or the following strings */
-#define         MR_CHATS_FOLDER             "Chats"      /* if we want to support Gma'l-labels - "Chats" is a reserved word for Gma'l */
+#define         MR_CHATS_FOLDER             "DeltaChats" /* Folder where the Chats will be moved to */
+#define         MR_CHATS_FOLDER_LEGACY      "Chats"      /* Folder name in old version. Will be renamed to new folder if present */
```

Comment:
i would also be fine in not converting the legacy folder:
- avoid additional levels of complexity and testing
- future changes may be more complicated
- it's no breaking change, just a new folder the legacy folder stays there and the user can move messages if wanted.
- Delta Chat is still beta